config/index.js

```diff
@@ -360,6 +360,38 @@ var conf = convict({
       doc: 'which key prefix to store sessions in redis under'
     }
   },
+  devicesCache: {
+    expiresIn: {
+      default: '1 hour',
+      format: 'duration',
+      env: 'CACHE_EXPIRES_IN',
+      doc: 'how long before devices cached expire'
+    },
+    generateTimeout: {
+      default: '2 seconds',
+      format: 'duration',
+      env: 'CACHE_GENERATE_TIMEOUT',
+      doc: 'how long hapi will wait for result when using cache before timing out'
+    },
+    enabled: {
+      default: true,
+      doc: 'Enable redis cache for devices',
+      format: Boolean,
+      env: 'USE_REDIS_FOR_DEVCES_CACHE'
```

Comment:
`DEVCES` Typo. Also see the note about prefixing  these env values`DEVICES_CACHE` above.